lib/ansible/playbook/play_context.py

```diff
@@ -74,6 +74,9 @@
     become_flags     = ('ansible_become_flags',),
     ssh_common_args  = ('ansible_ssh_common_args',),
     docker_extra_args= ('ansible_docker_extra_args',),
+    kubernetes_cluster = ('ansible_kubernetes_cluster',),
+    kubernetes_namespace = ('ansible_kubernetes_namespace',),
+    kubernetes_config_file = ('ansible_kubernetes_config_file',),
```

Comment:
we are trying to avoid poluting play_context this way,  we are looking to pass any `ansible_<pluginname>_.+`  vars to the plugin specifically instead.